IntegrationTests/IntegrationScenarios/Scenarios/SessionReplay/SRMultipleViewsRecordingScenarioTests.swift

```diff
@@ -78,7 +78,7 @@ class SRMultipleViewsRecordingScenarioTests: IntegrationTests, RUMCommonAsserts,
         })
 
         let rawSRRequests = try srEndpoint.pullRecordedRequests(timeout: dataDeliveryTimeout, until: {
-            try SRSegmentMatcher.segmentsCount(from: $0) == Baseline.totalSegmentsCount
+            return try SRSegmentMatcher.segmentsCount(from: $0) >= Baseline.totalSegmentsCount
```

Comment:
suggestion/ Depending how much flakiness we still anticipate here, we could utilize `sendCIAppLog()` API to notify on the number of pulled segments:
```swift
let segmentsCount = try SRSegmentMatcher.segmentsCount(from: $0)
sendCIAppLog("Pulled \(segmentsCount) segments")
return segmentsCount >= Baseline.totalSegmentsCount
```

It uses STDOUT under the hood (`print()`) which is intercepted by CI Visibility, so we can see it in test logs in CI App.